TimberAPI/CustomObjectSystem/CustomObjectCollection.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Timberborn.AssetSystem;
+using Timberborn.EntitySystem;
+using UnityEngine;
+
+namespace TimberbornAPI.CustomObjectSystem
+{
+    public class CustomObjectCollection : IObjectCollection
+	{
+
+		public IEnumerable<Object> GetObjects()
```

Comment:
Why does this actually need to exists ? since it only calls a static right away.

Reply:
I should make the variable private

Reply:
It is private - I want to make it only accessible via this